features/FxA-45-device-registration-api/FX-DESKTOP.md

```diff
@@ -0,0 +1,112 @@
+Firefox Device Registration
+===========================
+
+This is a breakdown of the states Firefox needs to handle/worry about.
+
+See the [README.md](README.md) file for a list of requirements and user
```

Comment:
I see no problem with this, but I do expect the devil to be hiding in some of the details :)